chain/client/src/test_utils/test_env_builder.rs

```diff
@@ -256,12 +264,32 @@ impl TestEnvBuilder {
             "Cannot set both num_shards and epoch_managers at the same time"
         );
         let ret = self.ensure_stores();
+
+        // TODO(#11265): consider initialising epoch config separately as it
+        // should be decoupled from the genesis config.
+        // However, there are a lot of tests which only initialise genesis.
+        let mut base_epoch_config: EpochConfig = (&ret.genesis_config).into();
```

Comment:
to double confirm, this block between 271-285 should be moved to each test and the test should build EpochConfigStore similarly and set it in the testloop builder (so line 305-316 will be the default one)?

Reply:
yes!